packages/Interface/src/DTK_C_API.cpp

```diff
@@ -301,6 +302,13 @@ void EvaluateFieldFunctionWrapper<double>(
 
 extern "C" {
 
+typedef enum {
+    DTK_SUCCESS = 0,
+    DTK_INVALID_HANDLE = -1,
+    DTK_UNINITIALIZED = -2,
+    DTK_UNKNOWN = -99
+} DTK_Error;
+
```

Comment:
If I understand correctly, `strerror` is only for the default `errno` values, and thus is useless to us.